.github/workflows/continous-integration.yml

```diff
@@ -39,4 +39,34 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Build with Gradle Wrapper
-        run: ./gradlew build --warning-mode=fail -Pmicronaut.runtime=${{ matrix.runtime }}
+        run: ./gradlew build --warning-mode=fail -Pmicronaut.runtime=${{ matrix.runtime }} -Pspotbugs.ignoreFailures=true
```

Comment:
why are you setting the property explicitly? isn't it true by default if not set (see your change in build.gradle)?